apps/connection/src/interfaces/connection.interfaces.ts

```diff
@@ -51,12 +51,12 @@ export interface IOrgAgentInterface {
   orgId: string;
 }
 
-export class IConnectionInterface {
-  connectionDto: ConnectionPayload;
+export class ICreateConnection {
```

Comment:
Please explain why are we using class instead of interface?

Reply:
I have changed class by interface.